src/controller/CHIPDeviceController.h

```diff
@@ -141,6 +141,13 @@ struct ControllerInitParams
      */
     bool removeFromFabricTableOnShutdown = true;
 
+    /**
+     * Specifies whether to utilize a storage fabric for initializing the DeviceController.
+     * If enabled and neither the operational key pair nor the NOC chain are provided, then
+     * attempt to locate a storage fabric corresponding to the provided storageFabricIndex.
```

Comment:
I think the "storage" bit is a bit of a red herring.  This locates a fabric and node identity identified by the given FabricIndex in the FabricTable.  How those got there (storage, some other mechanism) is not really relevant here, right?

Reply:
Yes..